js_modules/dagster-ui/packages/ui-core/src/search/useIndexedDBCachedQuery.tsx

```diff
@@ -139,7 +139,11 @@ export class IndexedDBQueryCache<TQuery, TVariables extends OperationVariables>
     const result = await this.queryFn(this.variables);
 
     if (result.data && !result.error) {
-      await this.cacheManager.set(result.data, this.version);
+      const dataToCache = result.data;
+      setTimeout(() => {
```

Comment:
If we wanted to be really fancy this could be a `requestIdleCallback`

Reply:
I was going to do that until I saw it's not supported in Safari.. I'm not sure if we polyfill it

Reply:
Daww that's no fun - yeah definitely not worth a polyfill!